target/CACHEDIR.TAG

```diff
@@ -0,0 +1,3 @@
+Signature: 8a477f597d28d172789f06886806bc55
```

Comment:
I included this file just to say that the /target directory should be excluded from git. If you look at the default .gitignore for a Rust project, /target is included in there.
https://github.com/github/gitignore/blob/main/Rust.gitignore